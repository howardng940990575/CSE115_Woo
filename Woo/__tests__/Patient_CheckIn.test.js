import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Patient_CheckinScreen from '../src/Patient_Checkin';

test('Patient Check In SnapShot', ()=> {
    const snap = renderer.create(<Patient_CheckinScreen />).toJSON();
    expect(snap).toMatchSnapshot();
});
