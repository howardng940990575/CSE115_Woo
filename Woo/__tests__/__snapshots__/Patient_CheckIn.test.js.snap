// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Patient Check In SnapShot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#72C3C9",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
     Check-in Screen Homepage
  </Text>
</View>
`;
